_data/chains/eip155-42069

```diff
@@ -0,0 +1,17 @@
+
+{
+    "name": "pegglecoin",
+    "chain": "42069",
+    "network": "mainnet",
+    "rpc": [],
+    "faucets": [],
+    "nativeCurrency": {
+      "name": "pegglecoin",
+      "symbol": "peggle",
+      "decimals": 18
+    },
+    "infoURL": "https://pegglecoin.com",
```

Comment:
this site seems to not exist

Reply:
Sorry was wrong site. Set it to our communities homepage.